spot-client/src/spot-tv/native-functions/native-controller/native-controller.js

```diff
@@ -0,0 +1,45 @@
+import { logger } from 'common/logger';
+
+/**
+ * Implements a controller that utilizes the event emitter of the native (Electron) process if available.
+ *
+ * Using it will enable SpotTVs running wrapped in Electron access OS-native functions.
+ */
+class NativeControntroller {
+    /**
+     * Some.
+     */
+    constructor() {
+        // eslint-disable-next-line
+        this.ipcRenderer = this.isAvailable() && __non_webpack_require__('electron').ipcRenderer;
+    }
+
+    /**
+     * Returns true if the native functions are available (electron env.), false otherwise (web env.).
+     *
+     * @returns {boolean}
+     */
+    isAvailable() {
+        return window && window.process && window.process.emit;
```

Comment:
I can a silly browser check utility somewhere that this might be better off stuffed in. I should probably re-use it for the electron desktop picker.

Reply:
Added there, thanks!